### PR TITLE
updates for data loading and minor cleaning

### PR DESCRIPTION
**Pull request by WraySmith**
- revised `board_games.csv` to include the columns I need for the 3D graph (x, y, z) and columns we won't be using from the file
- minor update/notes in app files

Recommend we leave `board_games_ss.csv` around for now

### CODE REVIEW

**Comment by mqharris** [comment]
`board_games_ss.csv` is modified from `board_games.csv` in the app_data?

Isn't the plan to switch to the updated dataset in `data/processed/bgg_with_names.csv` for milestone 4?

How long are you looking to keep `board_games_ss.csv` around for.

**Comment by WraySmith** [comment]
> `board_games_ss.csv` is modified from `board_games.csv` in the app_data?
> 
> Isn't the plan to switch to the updated dataset in `data/processed/bgg_with_names.csv` for milestone 4?
> 
> How long are you looking to keep `board_games_ss.csv` around for.

- `board_games_ss.csv` is a direct copy prior to modifications in the event the modifications resulted in any issues (didn't see any from app testing), suggest deleting after another PR.
- I think the workflow for the data folder for now should be: raw (downloaded data) >> processed (apply scripts, ideally automate) >> app_data (manual move of processed when we're 100% sure it is good)
- After we're confident that the files in processed are working, we can consider having it read directly from processed

**Comment by WraySmith** [comment]
It may also be worth keeping the `app_data` folder around if we want to keep some processed data that isn't used by the app so it would then be:: raw (downloaded data) >> processed (apply scripts, ideally automate) >> app_data (automated move but pairing down some of the data from processed to be what is directly used by the app)

File: src/app/app_graphing.py [hunk 1]
```diff
@@ -67,6 +67,8 @@ def scatter_plot_dates(data, col="category", list_=[None]):
         )
     )
 
+    # THIS NEEDS TO BE DONE OUTSIDE OF ALTAIR
+    # OR IDEALLY USE EXPIREMENTAL ALTAIR TRANSFORMER
```
Comment (mqharris) [inline src/app/app_graphing.py:71]: Its unclear what `this` refers to

Does it refer to subsetting the df to 2 columns?
Does it refer to the aggregate function on : ` .encode(x="year_published:T", y="mean(average_rating)")`?
Or something else?

In the first case pandas is doing the subsetting , not altair

File: src/app/app_graphing.py [hunk 2]
```diff
@@ -225,9 +229,11 @@ def top_n_plot(data, cat=[None], mech=[None], pub=[None], n=10):
 
     return: altair plot
     """
+    plot_data = app_wr.call_boardgame_filter(data, cat, mech, pub, n)
```
Comment (mqharris) [inline src/app/app_graphing.py:232]: Are you moving this into its own line for cleanliness?
Or for performance

I dont think this should make a different if youre looking for performance

File: src/app/app_graphing.py (hunk 1 above)
Comment (WraySmith) [reply src/app/app_graphing.py:71]: refers to `.encode(x="year_published:T", y="mean(average_rating)")`

File: src/app/app_graphing.py (hunk 2 above)
Comment (WraySmith) [reply src/app/app_graphing.py:232]: cleanliness

File: src/app/app_graphing.py (hunk 1 above)
Comment (mqharris) [reply src/app/app_graphing.py:71]: yeah makes sense

**Review by mqharris — APPROVED** [review]
lgtm

**Pull request merged**